index.js

```diff
@@ -1,31 +1,37 @@
 #!/usr/bin/env node
 
-'use strict'
+"use strict"
 
-const Clubhouse = require('clubhouse-lib')
-const { ArgumentParser } = require('argparse')
+const Clubhouse = require("clubhouse-lib")
+const { ArgumentParser } = require("argparse")
 const { Octokit } = require("@octokit/rest")
 const ghIssueRe = /#(\d+)/gi
 
 let parser = new ArgumentParser({
-  version: '1.0.0',
+  version: "1.0.0",
   addHelp: true,
-  description: 'Tilt exterminator tool'
-});
+  description: "Tilt exterminator tool",
+})
+
+parser.addArgument("--dry-run", {
+  help: "Print out what data will change rather than changing anything",
+  action: "storeTrue",
+})
 
 let subparsers = parser.addSubparsers()
-let syncParser = subparsers.addParser('sync', {
+let syncParser = subparsers.addParser("sync", {
   addHelp: true,
-  help: 'sync an issue from github to clubhouse',
+  help: "sync an issue from github to clubhouse",
 })
 
-syncParser.addArgument(['-i', '--issue'], {
-  help: 'The github issue number to import into clubhouse',
+syncParser.addArgument(["-i", "--issue"], {
```

Comment:
(this doesn't have to change now but) if the only use of this tool is to import an issue, shouldn't this just be a required positional arg and not a flag?